Added SignIn page(Static)

jawahar-linx30 opened the pull request:
- SignIn component added
- Small bug fixes in the home component

Commit:
Added SignIn page

- SignIn component added
- Small bug fixes in home component

PatelNeelMahesh commented on Frontend/src/components/SignIn.jsx:
```diff
@@ -1,9 +1,252 @@
+                                    backgroundColor="#4A4A4A"
+                                ></Box>
+                            </Flex>
+                            {<GoogleButton />}
```
Why have you kept `<GoogleButton />` in {} here?

jawahar-linx30 replied:
Hey Neel, it was first commented on. When uncommenting I forgot to remove the curly brackets and removed only the asterisks and slashes. It doesn't affect the code but it will look inconsistent. I'll remove that rn.

Commit:
Bug fix in SignIn.jsx

- removed unwanted curly braces to improve code consistency

Commit:
Google OAuth setup

- Google login enabled, user data can be fetched.
- Used environmental variable to store google client_id.
- User's details fetched using the access_token of the user and logged in console.
- Currently no redirect_uri. It'll be added after the dashboard component is ready.

Commit:
Revert "Google OAuth setup"

This reverts commit db50b6881312fbd3aac9f7f74437d4378410c819.
